packages/gtk3.rb

```diff
@@ -50,6 +51,7 @@ def self.build
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C build install"
+    system "sed -i 's,null,,g'  #{CREW_DEST_LIB_PREFIX}/pkgconfig/gtk*.pc"
```

Comment:
Should this be in a patch section?

Reply:
This is patching after install to the destdir, so it needs to happen after install but before package creation. Doesn't self.patch happen before build?

Reply:
It does but I just wanted to ask if it was possible to do it before the build.

Reply:
That would be ideal. But I have **no idea** where that came from in the build. This fixes subsequent builds, so I'm not spending any more headspace on it.